pkg/ovs/openflow/ofctrl_action.go

```diff
@@ -16,22 +16,21 @@ type ofFlowAction struct {
 
 // Drop is an action to drop packets.
 func (a *ofFlowAction) Drop() FlowBuilder {
-	dropAction := a.builder.Flow.Table.Switch.DropAction()
-	a.builder.ofFlow.lastAction = dropAction
+	a.builder.Drop()
 	return a.builder
 }
 
 // Output is an action to output packets to the specified ofport.
 func (a *ofFlowAction) Output(port int) FlowBuilder {
 	outputAction := ofctrl.NewOutputPort(uint32(port))
-	a.builder.ofFlow.lastAction = outputAction
+	a.builder.ApplyActions([]ofctrl.OFAction{outputAction})
```

Comment:
I did not see where ApplyActions is implemented. Is it in Antrea code? Might need you to explain this change for me.

Reply:
> Wenying, I did not mean to have network policy to allow connections between podA to podB. But from pod label X (corresponds to local pods A and B) to label Y.
> Current flow for established connections of egress rule table (#50):
> cookie=0x11000000000000, duration=17599.565s, table=50, n_packets=6857, n_bytes=671986, priority=210,ct_state=-new+est,ip actions=resubmit(,70)
> 
> I am presuming above OVS flow stats correspond to all network policy egress rules. To get separate statistics for each network policy egress rule. Will above rule broken into fine-grain rules to track the statistics using the new matches? Wondering if that is the intent?

In my opinion, these new fields are not used to to change current implementations for egress/ingress flow entries. The requirements for them are for the metrix of NetworkPolicy rule. So I would like to use another table(after egress/ingress rule table) specific for metrix to avoid flow confliction. @tnqn could you add your thoughts about how you expect to leverage these ct fields( ct_nw_xx/ct_tp_xx fields )?

Reply:
Ok. One minor thing - I saw in all your code, ApplyActions() always takes a single action. In this case, should we consider to add a func like ApplyAction() that adds a single action (instead of a slice)?

Reply:
Sure, I will add such functions in ofnet.